generate/genhcl/genhcl.go

```diff
@@ -31,24 +31,24 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
-// StackHCLs represents all exported terraform code for a stack,
-// mapping the exported code name to the actual Terraform code.
+// StackHCLs represents all generated HCL code for a stack,
+// mapping the generated code name to the actual HCL code.
```

Comment:
`mapping the generated code name` are we referring to the actual code content (like a blok name) or is it the filename?

Reply:
It is the filename, will improve this